ui/src/components/canvas/FlowCanvas.tsx

```diff
@@ -81,9 +88,31 @@ const FlowCanvas = () => {
     updateLayoutDensity,
   } = useAppActions()
 
+  const cntlZPressed = useKeyPress("Control+z")
```

Comment:
It looks like [key press hook](https://reactflow.dev/api-reference/hooks/use-key-press) supports passing in a list of combinations. So for example, we could do `useKeyPress(["Control+z", "Meta+z"])` rather than two separate calls of the hook.